content/blog/2021/11/deep-dive-into-policy-logging.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Deep Dive into policy logging
+authors:
+    - Rafael Fernández López
+date: 2021-11-15
+---
+
+Policies are regular programs. As such they have the need to log often times. In general, we are
```

Comment:
Nit:

```suggestion
Policies are regular programs. As such they often have the need to log timestamps. In general, we are
```

Reply:
I will change it to emit messages, but honestly that might mean so many different things. Emit messages to whom, where? While the act of logging is a well known action that comprises certain behaviors (e.g. write a message next to a timestamp, possibly with a component tag, possibly with metadata in a certain format...).

Reply:
Changed it to log information.